releasenotes-dca/notes/ad-data-race-cluster-checks-provider-2b1a6a1cb781b296.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG-DCA.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    Fix data race in autodiscovery cluster checks provider.
```

Comment:
```suggestion
    Fix data race in the Autodiscovery cluster checks provider.
```

Reply:
since this is backport, leaving it as it was in the original PR